assets/csidriveroperators/aws-ebs/01_namespace.yaml

```diff
@@ -5,3 +5,5 @@ metadata:
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
     openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
```

Comment:
We should add the label [in this file](https://github.com/openshift/cluster-storage-operator/blob/master/manifests/02_csi_driver_operators_namespace.yaml), since it's CVO who creates this namespace.

In theory we could remove these asset files from here.

Reply:
Hold on, why do we create the namespace unconditionally? It has no use on bare metal.

Reply:
from git log:
```
Let CVO create namespace for CSI driver operators.
   
So that the openshift-cluster-csi-drivers namespace exists
when the cloud-credential-operator deploys the secret.
```
Good catch! I removed the namespace creation from all CSI driver operators.